index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <style type="text/css">
+        @media screen and (min-width: 400px) {
+        
+      }
+        
+        </style>
```

Comment:
Kindly use class instead of id in this case. This will help you style all the similar parts of HTML at once.